package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "test": "jest --coverage",
     "start": "nodemon Server/server.js",
-    "dev:react": "webpack -d --watch",
-    "db:setup": "Database/seed.js"
+    "docker": "webpack -p && node database/seed.js & node Server/server.js",
+    "dev:react": "webpack -p"
   },
```

Comment:
It seems the script "dev:react" is for developer settings, but the command flag is for production.